packages/config/src/common/ProjectRiskView.ts

```diff
@@ -0,0 +1,14 @@
+import { ProjectRiskViewEntry } from './ProjectRisk'
+
+export interface ProjectRiskView
+  extends Required<{
+    validatedBy?: ProjectRiskViewEntry
+    sourceUpgradeability?: ProjectRiskViewEntry
+    destinationToken?: ProjectRiskViewEntry
+  }> {
```

Comment:
why
```tsx
Required<{
    validatedBy?: ProjectRiskViewEntry
    sourceUpgradeability?: ProjectRiskViewEntry
    destinationToken?: ProjectRiskViewEntry
  }>
```

instead of adding this to the interface directly?

```tsx
{
    validatedBy: ProjectRiskViewEntry
    sourceUpgradeability: ProjectRiskViewEntry
    destinationToken: ProjectRiskViewEntry
}
```